plugins/provenance/server/resource.py

```diff
@@ -323,8 +325,10 @@ def snapshotResource(self, obj):
         return snap
 
     def fileSaveHandler(self, event):
-        """When a file is saved, update the provenance of the parent item.
-        :param event: the event with the file information."""
+        """
+        When a file is saved, update the provenance of the parent item.
+        :param event: the event with the file information.
```

Comment:
Agreed, but that seems like a separate, larger issue. That blank line is missing in every method in this file, as well as in other places in this project.

Could documentation generation be covered by a test? Running sphinx shows some errors and warnings, and I don't think it even processes the plugin code right now.